### PR TITLE
chore: improve event dashboard modals wording

### PR DESCRIPTION
**Pull request by gikf**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

---

- More descriptive wording in modals in event dashboards.

### CODE REVIEW

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Comment by Sboonny** [comment]
what do you think of changing the Share text to `Copy link`, I have used share because I was planning to create a button that share in twitter, linkendin or stuff alike.

But now it only copy the link text

File: client/src/modules/dashboard/Events/pages/EventPage.tsx [hunk 1]
```diff
@@ -65,14 +65,22 @@ export const EventPage: NextPageWithLayout = () => {
   const onConfirmAttendee =
     ({ eventId, userId }: MutationConfirmAttendeeArgs) =>
     async () => {
-      const ok = await confirm();
+      const ok = await confirm({
+        title: 'Confirm attendee?',
+        body: 'Are you sure you want to confirm attendee?',
```
Comment (ojeytonwilliams) [inline client/src/modules/dashboard/Events/pages/EventPage.tsx:70]:
```suggestion
        body: 'Are you sure you want to confirm the attendee?',
```

File: client/src/modules/dashboard/Events/pages/EventPage.tsx [hunk 2]
```diff
@@ -65,14 +65,22 @@ export const EventPage: NextPageWithLayout = () => {
...
       if (ok) confirmAttendee({ variables: { eventId, userId } });
     };
 
   const onRemove =
     ({ eventId, userId }: MutationDeleteAttendeeArgs) =>
     async () => {
-      const ok = await confirmDelete();
+      const ok = await confirmDelete({
+        buttonText: 'Remove user',
+        body: 'Are you sure you want to remove user from event?',
```
Comment (ojeytonwilliams) [inline client/src/modules/dashboard/Events/pages/EventPage.tsx:81]:
```suggestion
        body: 'Are you sure you want to remove this user from the event?',
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Just grammar nitpicks:

File: client/src/modules/dashboard/Events/pages/EventPage.tsx (hunk 2 above)
Comment (ojeytonwilliams) [reply client/src/modules/dashboard/Events/pages/EventPage.tsx:81]: I couldn't tell you why "Remove user from event?" is fine, while "Are you sure you want to remove user from event?" is not. English grammar makes no sense :man_shrugging:

**Review by ojeytonwilliams — APPROVED** [review]
LGTM 👍

**Pull request merged**